server/index.js

```diff
@@ -98,4 +100,85 @@ app.get('/system_req/platforms/:product_id', (req, res) => {
   });
 });
 
+//Extended CRUD for SDC
+
+//GET
+app.get('/readOnly/:product_id', (req, res) => {
+  const id = req.params.product_id;
+
+  if (id > 100 || id < 1) {
+    console.log('Product id must be 1-100 inclusive. Invalid product_id: ', id);
+    res.status(404).send();
+  } else {
+    Overview.find({ product_id: id })
+      .then(doc => {
+        const productInfo = doc;
+        res.send(productInfo);
+      })
+      .catch(err => {
+        console.log('error in GET readOnly: ', err);
+        res.status(404).send(err);
+      })
+  }
+});
+
+
+//POST
+app.post('/newItem', (req, res) => {
+  const newItem = req.body;
+
+  Overview.create(newItem)
+    .then(doc => {
+      const productInfo = doc;
+      res.status(201).send(productInfo);
```

Comment:
I've never looked into status code 201. That's really good. I will change mine for 201. Thanks!